libraries/AC_Fence/AC_Fence.h

```diff
@@ -194,6 +201,8 @@ class AC_Fence
     AP_Int8         _total;                 // number of polygon points saved in eeprom
     AP_Int8         _ret_rally;             // return to fence return point or rally point/home
     AP_Int16        _ret_altitude;          // return to this altitude
+    AP_Int8         _alt_frame;             // frame that alt_max and alt_min are currently in
+    AP_Int8         _alt_frame_ext;         // front facing alt frame
```

Comment:
`_alt_frame` is not a parameter so should be a `Int8_t` type

Reply:
I don't think we should need the _alt_frame_ext parameter actually.